pkgs/development/python-modules/pyro4/default.nix

```
@@ -32,10 +32,15 @@ buildPythonPackage rec {
   ];
 
   checkInputs = [ pytestCheckHook ];
+
   # add testsupport.py to PATH
   preCheck = "PYTHONPATH=tests/PyroTests:$PYTHONPATH";
+
   # ignore network related tests, which fail in sandbox
-  pytestFlagsArray = [ "--ignore=tests/PyroTests/test_naming.py" ];
+  pytestFlagsArray = [ "--ignore=tests/PyroTests/test_naming.py" ]
+  # test hangs on darwin with sandbox enabled
+    ++ lib.optionals stdenv.isDarwin [ "--ignore=PyroTests/test_daemon.py" ];
```

Maybe it is possible to use `disabledTests` instead?

```suggestion
  disabledTests = [
    "test_naming"
  ] ++ lib.optionals stdenv.isDarwin [
    "test_daemon"
  ];
```

---

disabledTests can only disable single tests but we are ignoring entire files here with multiple tests. See https://github.com/irmen/Pyro4/blob/4.80/tests/PyroTests/test_daemon.py